lib/livebook/application.ex

```diff
@@ -179,6 +180,20 @@ defmodule Livebook.Application do
     defp app_specs, do: []
   end
 
+  if Livebook.Config.feature_flag_enabled?(:hub) do
```

Comment:
Let's define a separate feature flag for it, because eventually we will remove the hub flag, but the local-host flag will likely stay forever.

Reply:
What if we just use `Mix.env() == :dev` (and, in this case, + feature flag verification)?

Reply:
Mix.env() == :dev is a anti-pattern, so we should avoid it. I also think the local-host hub will be useful for testing. :)